erpnext/accounts/doctype/accounting_dimension/test_accounting_dimension.py

```
@@ -79,6 +81,114 @@ def test_mandatory(self):
+
+	def test_client_server_fieldname_consistency(self):
+		"""Test that client-side JS and server-side Python generate the same fieldname."""
+		test_cases = [
+			("Département", "departement"),
+			("Département Region", "departement_region"),
+			("Zone-Test", "zone_test"),
+			("Café-Restaurant München", "cafe_restaurant_munchen"),
+			("Área de Vendas", "area_de_vendas"),
+		]
+
+		for label, expected_fieldname in test_cases:
```

Ruff (0.15.7)

[warning] Loop control variable `expected_fieldname` not used within loop body

Rename unused `expected_fieldname` to `_expected_fieldname`

(B007)